ycast/radiobrowser.py

```diff
@@ -61,14 +64,15 @@ def request(url):
 
 
 def get_station_by_id(uid):
-    station_json = request('stations/byid/' + str(uid))
+    station_json = request('stations/byuuid/' + str(id_registry[uid]))
```

Comment:
This is already fixed in https://github.com/milaq/YCast/pull/77

Reply:
Ok then